contrib/aktualizr-toml-update

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh -e
 
+#Defines the SOTA Client systemd service name.
```

Comment:
I believe we don't need this comment. The variable name is self-descriptive.

Reply:
@vkhoroz I was testing one idea but didn't work. I was trying to simplify and avoid to create a template file but didn't work.
I was using `sed -i `  to avoid copy the file but this is not a good idea. On yocto side if someone is running a local build and change the `SOTA_CLIENT` after the first build it will not update.

Reply:
indeed. you are correcty. Thank you for the clarification.